nimbus-features/messagingFeature.yaml

```diff
@@ -56,6 +56,8 @@ features:
             AFTER_THREE_LAUNCHES_THIS_WEEK: app_cycle.foreground|eventSum('Weeks', 1, 0) >= 3
             USER_RECENTLY_INSTALLED:  days_since_install < 7
             USER_RECENTLY_UPDATED:    days_since_update < 7 && days_since_install != days_since_update
+            USER_INSTALLED_OR_UPDATED_YESTERDAY: days_since_update >= 1
```

Comment:
can we clarify the name of the trigger? To something that captures we are > 1 day since install or update, as opposed to 'the user updated or installed yesterday'. Those would be different things

Reply:
Renamed it to `MORE_THAN_24H_SINCE_INSTALLED_OR_UPDATED`.